### PR TITLE
Add pipeline for production cdn

### PR DESCRIPTION
**Pull request by tjololo**
## Description
Publish changes to production cdn

## Related Issue(s)
- #{issue number}

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [x] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: .github/workflows/deploy-to-production-cdn.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Deploy files to staging cdn
```
Comment (altinnadmin) [inline .github/workflows/deploy-to-production-cdn.yaml:1]: Should name be to production?

File: .github/workflows/deploy-to-production-cdn.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+on:
+  push:
+    branches:
+    - master
```
Comment (altinnadmin) [inline .github/workflows/deploy-to-production-cdn.yaml:6]: Should we rename to branch to main before going to prod? Or is the timing for doing that irrelevant?

**Review by altinnadmin — APPROVED** [review]
LGTM, two small comments to consider :)

File: .github/workflows/deploy-to-production-cdn.yaml (hunk 2 above)
Comment (tjololo) [reply .github/workflows/deploy-to-production-cdn.yaml:6]: I think that is irrelevant, small change in the pipeline. There are other external pipelines that push to master so we need to sync that change with them

File: .github/workflows/deploy-to-production-cdn.yaml (hunk 1 above)
Comment (tjololo) [inline .github/workflows/deploy-to-production-cdn.yaml:1]:
```suggestion
name: Deploy files to production cdn
```

**Pull request merged**